src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -98,9 +98,24 @@ def delete_fs_volume(self, volname, confirm):
                 "that is what you want, re-issue the command followed by " \
                 "--yes-i-really-mean-it.".format(volname)
 
+        ret, out, err = self.mgr.check_mon_command({
```

Comment:
@kotreshhr tests are failing now with,
```
AttributeError: 'Module' object has no attribute 'check_mon_command'
```

`check_mon_command` is not in nautilus.  Is it OK just to use `mon_command` instead?

Reply:
Yes, mon_command can be used. The only difference between them is that 'check_mon_command' raised if ret !=0

Reply:
As discussed that'd raise an unhandled exception. If ret !=0 I just return the output of mon_command().

Reply:
lgtm